gwpy/plot/colorbar.py

```diff
@@ -69,24 +59,8 @@ def process_colorbar_kwargs(figure, mappable=None, ax=None, use_axesgrid=True,
     # -- create axes for colorbar (if required)
 
     cax = kwargs.pop('cax', None)
-    if cax is None:
-        # show warning about pending change
-        #     current default is to shrink the axes a wee bit,
-        #     which will change to no shrinking before 1.0
-        if kwargs.get('fraction') is None:
-            warnings.warn("`fraction` was not specified. Currently the "
-                          "default is `0.15`, matching the previous release "
-                          "behaviour, however, this will change to `0.0` in "
-                          "an upcoming release. To keep the current default, "
-                          "manually specify `fraction=0.15`",
-                          PendingDeprecationWarning)
-            kwargs['fraction'] = 0.15
-
-        # create axes if requesting not to resize the parent
-        if use_axesgrid:
-            cax, kwargs = make_axes_axesgrid(ax, **kwargs)
-    else:
```

Comment:
@duncanmmacleod Is there a reason that this scenario (cax is not None) is no longer handled in the modified code?